docs/python/environments.md

```diff
@@ -4,39 +4,40 @@ Area: python
 TOCTitle: Environments
 ContentId: 8fe4ca8b-fc70-4216-86c7-2c11b6c14cc6
 PageTitle: Using Python Environments in Visual Studio Code
-DateApproved: 10/05/2021
+DateApproved: 03/20/2022
 MetaDescription: Configuring Python Environments in Visual Studio Code
 MetaSocialImage: images/tutorial/social.png
 ---
 # Using Python environments in VS Code
 
-This article discusses the helpful features provided by the VS Code Python extension for working with Python environments. An "environment" in Python is the context in which a Python program runs and consists of an interpreter and any number of installed packages. After you finish this article, you'll have a good understanding of:
+This article discusses the helpful Python environments features available in VS Code. An "environment" in Python is the context in which a Python program runs and consists of an interpreter and any number of installed packages.
 
-- [General environment concepts](#global-and-virtual-environments)
-- [How to select an environment](#select-and-activate-an-environment)
-- How to create a [virtual](#create-a-virtual-environment) or [conda](#create-a-conda-environment) environment
-- How to [select and activate](#select-and-activate-an-environment) a virtual environment
-- How to work with [Environments and Terminal windows](#environments-and-terminal-windows)
-- [Where the Python extension looks for environments](#where-the-extension-looks-for-environments)
-- Environment variables and [environment variable definitions files (.env)](#environment-variable-definitions-file)
+## Python environments
 
-While this article provides some information about Python environments and their concepts, it is primarily for understanding how to work with them within VS Code. If you're new to working with Python environments, you can learn more at [Virtual Environments and Packages (Python.org)](https://docs.python.org/3/tutorial/venv.html) and [Installing Python Modules (Python.org)](https://docs.python.org/3/installing/index.html#installing-index).
+### Global environments
 
-> **Note**: If you're looking to get started with Python in Visual Studio Code, refer to the tutorial [Getting Started with Python in VS Code](/docs/python/python-tutorial.md). The [Python tutorial (Python.org)](https://docs.python.org/3/tutorial/index.html) might also be helpful if you're new to the Python language.
+By default, any Python interpreter installed runs in its own **global environment**. They aren't specific to a particular project. For example, if you just run `python` (Windows) or `python3` (macOS/Linux) at a new command prompt, you're running in that interpreter's global environment. If Python is not added to PATH, run `py` (Windows) at a new command prompt instead. Any packages that you install or uninstall affect the global environment and all programs that you run within it.
```

Comment:
It can also be just `python3` on Windows as well.

```suggestion
By default, any Python interpreter installed runs in its own **global environment**. They aren't specific to a particular project. For example, if you just run `python`, `python3`, or `py` at a new terminal (depending on how you installed Python), you're running in that interpreter's global environment. Any packages that you install or uninstall affect the global environment and all programs that you run within it.
```